examples/sdk-app-example/README.md

```diff
@@ -3,8 +3,8 @@
 Includes build variants for both Java and Kotlin
 
 1. Open this project in Android Studio
-2. Insert your API key into the manifest placeholder for `bugsnagApiKey` in
-   `build.gradle`.
+2. Insert your API key into the AndroidManifest.xml
 3. Select build variant (either `javaExample` or `kotlinExample`)
-4. Run the app! There are several examples of different kinds of errors which
+4. Build the `ndk` module (select it from the menu)
+5. Run the app! There are several examples of different kinds of errors which
```

Comment:
No gradle command is required if you build the NDK module first

Reply:
While true, this won't be a problem once either:

1. We ship a release and update the example to use external dependencies. The core issue is that the bugsnag *.so files aren't unpacked into place automatically because the gradle plugin doesn't recognize where the bugsnag-ndk aar is.
2. We update the gradle plugin to handle these kinds of edge cases, where bugsnag-ndk comes from somewhere other than an archive explicitly named "bugsnag-android-ndk". I've made a start as to what this future logic may look like in the example build.gradle, though it needs to be properly designed.

It doesn't seem unreasonable that in the meantime, while testing this pull request, there is an extra step to run the example project.